qiskit_dynamics/solvers/solver_utils.py

```diff
@@ -35,17 +35,15 @@
 
 def is_lindblad_model_vectorized(obj: any) -> bool:
     """Return True if obj is a vectorized LindbladModel."""
-    return isinstance(obj, LindbladModel) and ("vectorized" in obj.evaluation_mode)
+    return isinstance(obj, LindbladModel) and obj.vectorized
 
 
 def is_lindblad_model_not_vectorized(obj: any) -> bool:
     """Return True if obj is a non-vectorized LindbladModel."""
-    return isinstance(obj, LindbladModel) and ("vectorized" not in obj.evaluation_mode)
+    return isinstance(obj, LindbladModel) and not obj.vectorized
```

Comment:
```suggestion
    return not is_lindblad_model_vectorized(obj)
```

Reply:
These are not actually the same. If `obj` is a `HamiltonianModel`, then 
```
(isinstance(obj, LindbladModel) and not obj.vectorized) == False
```
but
```
not is_lindblad_model_vectorized(obj) == True
```

`obj` needs to be a `LindbladModel` for both of these functions to return `True`.

Reply:
Certainly, you're right.